app/models/user_team.rb

```diff
@@ -77,6 +77,15 @@ def destroy(new_owner)
       protocol.save
     end
 
+    # Make new owner author of all inventory items that were added
+    # by departing user and belong to this team.
+    repository_rows =
+      RepositoryRow.joins(:repository)
+                   .where('repositories.team_id = ? and repository_rows.created_by_id = ?', team, user)
+    repository_rows.find_each do |repository_row|
+      repository_row.update(created_by: new_owner)
```

Comment:
You could avoid this loop by calling `repository_rows.update_all(created_by: new_owner)`

Reply:
Could you move this functionality to RepositoryRow (into new method change_owner)